test/hpke_test.c

```diff
@@ -0,0 +1,1410 @@
+/*
+ * Copyright 2022 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <openssl/evp.h>
+#include <openssl/core_names.h>
+#include <openssl/rand.h>
+#include <openssl/hpke.h>
+#include "testutil.h"
+
+#ifndef OSSL_HPKE_MAXSIZE
```

Comment:
You could just use another constant :)

Reply:
did s/OSSL_HPKE_MAXSIZE/OSSL_HPKE_TXTSIZE/g